bindings/core/src/method/mod.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod account;
+mod client;
+mod secret_manager;
+mod utils;
+mod wallet;
+
+pub use self::{account::AccountMethod, client::ClientMethod, utils::UtilityMethod, wallet::WalletMethod};
```

Comment:
Why not re-export `SecretManagerMethod` ?

Reply:
Because I only did the `SecretManagerMethod` yet, nothing else
Not even 100% sure if we want it

Reply:
Added